.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+	"extends": [
+		"@nextcloud"
+	],
+	"ignorePatterns": ["node_modules", "dist"],
+	"overrides": [
+		{
+			"files": ["tests/*.js"],
+			"rules": {
+				"n/no-unpublished-import": "off",
+				// Todo: convert tests to TS then drop overrides below
+				"n/no-missing-import": "off",
+				"import/no-unresolved": "off",
+				"import/extensions": "off"
+			}
+		},
+		{
+			"files": ["*.config.*js"],
+			"rules": {
+				"n/no-unpublished-import": "off"
+			}
+		}
+	]
```

Comment:
Would be amazing to upstream that to the esling config :rocket:

Reply:
`"cypress/**/*.js",  "cypress/**/*.ts", "*.cy.js", "*.cy.ts"` too (cypress)
`"*.config.*js", "*.config.*ts"`